tests/dependency-convergence/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.google.cloud</groupId>
   <artifactId>full-convergence-check</artifactId>
   <packaging>pom</packaging>
-  <version>0.175.0</version><!-- {x-version-update:google-cloud-bom:current} -->
+  <version>0.176.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-bom:current} -->
```

Comment:
The current value for google-cloud-bom is "0.176.1-SNAPSHOT", but Release Please didn't update it because the tests module was not managed.